### DISCUSSION TITLE
Add marked.esm, marked.js to git.ignore?

### DISCUSSION THREAD

**Discussion by calculuschild** [markedjs/marked#1898] [Ideas]
My understanding is we don't want to have marked.esm, marked.js, or the other compiled files pushed in PRs generally? Would it make sense to officially just add them from the git.ignore file? I'm not a Git wizard so I'm not sure what the best practice is, but I find myself constantly almost submitting commits that include them.

**Comment by UziTech** [comment]
We can't remove them from the git repo since some users import them client side using 
`<script src="https://cdn.jsdelivr.net/gh/markedjs/marked/lib/marked.js"></script>`

It should be fine to include them in PRs since they are automatically overwritten once merged.

In fact if they are included in a PR you can test that PR in our [demo](https://marked.js.org/demo/)

![image](https://user-images.githubusercontent.com/97994/104034267-446efa00-5196-11eb-85ab-c671947371cf.png)

**Reply by calculuschild** [reply]
!!!!! I was wondering why my PRs weren't working in the demo!!! I just assumed it was being compiled in the background with the deployment bot.

**Reply by UziTech** [reply]
Ya it worked until we broke the files up into modules. It would be nice to figure out how to make it work again.
